### PR TITLE
screen: fix garbled statusline in Big Sur

### PR DESCRIPTION
**Pull request by arjanvandervelde**
add include to configure.ac to fix check for getloadavg(). this fixes issue https://trac.macports.org/ticket/61919

###### Type(s)

- [x] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS x.y
Xcode x.y

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [ ] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

**Comment by arjanvandervelde** [comment]
I've added includes for the remaining implicit declarations; this fixes "you don't have jobcontrol" message during configure and makes it properly detect pty-related functions.

**Comment by dgilman** [comment]
LGTM, although I would appreciate if someone could retrigger the macos 11 build so that can be clean before merging. I will see about sending this patch upstream too.

**Comment by arjanvandervelde** [comment]
awesome, thanks!

**Comment by arjanvandervelde** [comment]
it looks like it ran okay the second time.

**Comment by dgilman** [comment]
I don't have permissions to close PRs, we have to wait for a team member to do so. I was just giving my consent earlier in hope it would accelerate the process.

**Comment by michaelld** [comment]
I've triggered the CI to redo itself. Let's let it run & then I'll merge. Thanks for your contribution & patience!

**Review by michaelld — APPROVED** [review]
LGTM ; waiting for CI ...

**Comment by arjanvandervelde** [comment]
I see, thank you! 👍

**Pull request merged**
